src/components/ThirdPartyCollectionDetailPage/CollectionContextMenu/CollectionContextMenu.module.css

```diff
@@ -6,3 +6,11 @@
 .ellipsis {
   margin: auto !important;
 }
+
+:global(.ui.dropdown).dropdown :global(.menu > .disabled.item) {
+  pointer-events: auto;
```

Comment:
Without this, the `Popup` won't be triggered on a disabled element since the `pointer-events: none` applied on the element will prevent the `:hover` action and so the `Popup` won't be rendered.

Reply:
Just tried it, seems you're right!

Now, what about adding these two styles to the `/src/themes/components/Dropdown.css` file, which is the place where we add global, component-based, overrides/styles.
We can remove it from [src/components/ThirdPartyCollectionDetailPage/CollectionItem/CollectionItem.module.css](https://github.com/decentraland/builder/pull/1800/files#diff-ac267dde608f7eeefb22ccb486995c649a76939701ea60c081832101c8507b08) too, and any other place where this was necessary.

NOW, i'm aware that this might be a bit dangerous. We need to do a quick check to see nothing broke. If you're not comfortable with that. This is fine

Reply:
Yeah good idea! I created a ticket to track that specifically and make sure it won't break any of the existing stuff :) 
#1810